src/api/consumer/lib/export.ts

```diff
@@ -177,12 +177,24 @@ async function getComponentsToExport(
 
 function getIdsWithFutureScope(ids: BitIds, consumer: Consumer, remote?: string | null): BitIds {
   const workspaceDefaultScope = consumer.config.workspaceSettings.defaultScope;
+  let workspaceDefaultOwner = consumer.config.workspaceSettings.defaultOwner;
+  // For backward computability don't treat the default binding prefix as real owner
+  if (workspaceDefaultOwner === DEFAULT_BINDINGS_PREFIX) {
+    workspaceDefaultOwner = undefined;
+  }
+
   const idsArray = ids.map(id => {
     if (remote) return id.changeScope(remote);
     if (id.hasScope()) return id;
     const overrides = consumer.config.componentsConfig?.getOverrideComponentData(id);
     const componentDefaultScope = overrides ? overrides.defaultScope : null;
-    return id.changeScope(componentDefaultScope || workspaceDefaultScope || null);
+    // TODO: handle separation of owner from default scope on component
```

Comment:
There are many "TODO". consider adding them to the original issue and provide link here.

Reply:
That's will be handled once we decide about the product of override owner / scope on a specific component. it will be handled under a new issue.

Reply:
https://github.com/teambit/bit/issues/2648